modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -88,27 +116,93 @@ data:
         basicAuth:
           username:
             name: remoteWriteAuth
-            key: user
+            key: <username>
           password:
             name: remoteWriteAuth
-            key: password
+            key: <password>
+----
++
+* bearer token 
++
+A bearer token (`Bearer`) configuration combines a basic authentication username and password with authorization credentials configured to use a specified token (such as an API key).
++
+The following shows a `Bearer` authorization sample that uses a `basicAuth` username and password combined with an authorization token.
+It assumes you have already created a `Bearer` secret with the name `rw-auth` in the `openshift-monitoring` namespace.
+Substitute your username, password, and token for the appropriate `key` values. 
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write.endpoint"
+      - url: "https://basicAuth.remotewrite.com/api/write"
+        basicAuth:
+          username:
+            name: rw-auth
+            key: <username>
+          password:
+            name: rw-auth
+            key: <password>
+      - url: "https://authorization.remotewrite.com/api/write"
+        authorization:
+          type: Bearer
+          credentials:
+            name: rw-auth
+            key: <token_name>
 ----
+* OAuth 2.0
++
+An OAuth 2.0 (`oauth2`) configuration uses the client credentials grant type. Prometheus fetches an access token from the specified remote write endpoint with the specified client access and secret keys.
```

Comment:
Prometheus fetches an access token from `tokenUrl`  with the specified client name and client secret, in order to access the remote write endpoint using this token.

Reply:
Updated the content per your comment